nemo/collections/common/parts/preprocessing/collections.py

```diff
@@ -1548,24 +1546,55 @@ def __parse_item_rttm(self, line: str, manifest_file: str) -> Dict[str, Any]:
             raise ValueError(
                 f"Manifest file has invalid json line " f"structure: {line} without proper audio file key."
             )
+
+        # Audio file handling
         if isinstance(item['audio_file'], list):
-            item['audio_file'] = [os.path.expanduser(audio_file_path) for audio_file_path in item['audio_file']]
+            audio_file_list = []
+            for single_audio_file in item['audio_file']:
+                if not os.path.exists(single_audio_file):
```

Comment:
I think we can skip checking file existence here, since it will check again in `AudioSegment.from_file()`. If you do want to keep it, we need to move it after calling `get_full_path` so that the relative paths are converted to absolute paths before checking

Reply:
If `get_full_path` is based on `AudioSegment.from_file()`, no need for checking steps.
Removed and driectly reading with `get_full_path`